test/test_helper.rb

```diff
@@ -22,6 +22,12 @@ class ActiveSupport::TestCase
   set_fixture_class({ :hosts => Host::Base })
 end
 
-def set_session_user
+def set_session_user2
   SETTINGS[:login] ? {:user => User.admin.id, :expires_at => 5.minutes.from_now} : {}
 end
+
+def set_session_user_reader
```

Comment:
is this used anywhere? the tests you changed are using the one in 'test_helper_discovery' right?